src/services/side-panels-manager/models/item-wrapper.tsx

```diff
@@ -1,15 +1,17 @@
 import { h, createRef, RefObject, FunctionalComponent, ComponentClass } from 'preact';
-import { KalturaPlayer } from 'kaltura-player-js';
+import { KalturaPlayer, PlaykitJS } from 'kaltura-player-js';
```

Comment:
where is PlaykitJS set ?

Reply:
@JonathanTGold 
@SivanA-Kaltura 

we do not need to use '@playkit-js/kaltura-player-js'?